recipes/dkist/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "dkist" %}
+{% set version = "1.0.0b11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dkist-{{ version }}.tar.gz
+  sha256: 54dc4a6bd60222b39f3ceb9a1e6a874b3ab2ec7466205ba48a98ff9a9ac81e04
+
+build:
+  skip: true  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - setuptools >=30.3.0
+    - setuptools-scm
+    - wheel
+    - pip
+  run:
+    - python
+    - appdirs >=1.4
+    - asdf >=2.9.1
+    - asdf-astropy
+    - asdf-transform-schemas
+    - astropy >=5
+    - dask-core >=2
+    - globus-sdk >=3.0
+    - gwcs >=0.18.0
+    - matplotlib-base >=3.1
+    - ndcube >=2.0
+    - numpy >=1.21
+    - parfive >=1.3
+    - sunpy >=4
+
+test:
+  requires:
+    - pip
+    - pytest
+    - pytest-mock
+    - pytest-mpl
+    - pytest-httpserver
+    - pytest-astropy
+    - hypothesis
+  imports:
+    - dkist
+  commands:
+    - pip check
+    - pytest --pyargs dkist
+
+about:
+  home: http://dkist.nso.edu
+  summary: DKIST User Tools
+  license: BSD-3-Clause
+  license_file: licenses/LICENSE.rst
+
+extra:
+  recipe-maintainers:
+    - Cadair
+    - SolarDrew
```

Comment:
@SolarDrew please confirm you are ok with co-maintaining this :smile:

Reply:
Yes, fine by me.